bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_10/A5_10_18.java

```diff
@@ -25,4 +27,19 @@ public class A5_10_18 extends A5_10 {
     public A5_10_18(ERP1Message packet) {
         super(packet);
     }
+
+    @Override
+    protected double getMaxUnscaledValue() {
+        return 250.0;
+    }
+
+    @Override
+    protected double getTempScalingFactor() {
+        return -6.25; // 250/40
```

Comment:
Perhaps simply:
```suggestion
        return -250 / 40;
```

If 250 is the max unscaled value, this could be extracted to a constant for being reused here and in `getMaxUnscaledValue` which perhaps would make it even more descriptive:
```java
private static final double MAX_UNSCALED_VALUE = 250;
```

```suggestion
        return -MAX_UNSCALED_VALUE / 40;
```

Reply:
Thanks a lot for your suggestions. I have generalized the temerature calculation now in the base class A5_10. I think it should not be neccessary to define consts now.